torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -873,8 +875,7 @@ def _post_backward_final_callback(
             torch.cuda.current_stream().synchronize()
     state._exec_order_data.next_iter()
 
-    states = [state] if _is_composable(state) else _get_fsdp_states(state)
-    for state in states:
+    for state in _get_fsdp_states(module):
```

Comment:
hmm, is this intentional to abuse the var name state?

Reply:
and on line 887, is it a bug? I assume your intention is to set it on the original `state` var instead of the one used in the loop?

Reply:
Oh man. Thanks for the great catch. Let me fix this.